contracts/currency-network/CurrencyNetworkInterface.sol

```diff
@@ -23,10 +43,21 @@ interface CurrencyNetworkInterface {
     )
         external;
 
+    /**
+     * @notice returns what _b owes to _a
+     * @param _a First address that defines the trustline
+     * @param _b second address that defines the trustline
```

Comment:
Not consistent with params `_from` and `_to`. The compilation fails therefore with
```
DocstringParsingError: Documented parameter "_b" not found in the parameter list of the function.
```